README.md

```diff
@@ -1 +1,75 @@
-# Capstone-React
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white) ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white) ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+# Marvel's Characters: 🦸🏻‍♂️🦸🏻‍♀️⚡
+- Microverse - React/Redux final capstone project
+
+## Acces link
+> [click here to check live version](https://capstone-react-vitorguedesmadeira.netlify.app/)
+
+## Tech ⌘⇧
+- **HTML & CSS best practices:** Correct use of tags, elements, properties and syntax.
+- **Gitflow:**  Correct use of Branches for deployment and features development.
+- **REACT:**  Used to set the basic structure.
+- **REDUX:** Centralizing application's state.
+- **MARVEL-API:** Fetching data.
+- **Webpack:**  Source development for production bundling.
+- **Linters Check:** Local and Pull Request Github workflows to check for stylistic errors, bugs and syntax errors in code.
+
+## Quick view
+
+> ![](/src/Images/desktopversion1.png)
+> ![](/src/Images/desktopversion2.png)
+
+## Video description
+
+> [click here to check a video description of the project](https://www.loom.com/share/ddad583acc2d498f996519af4e3815e7
+)
+
+## Setup
+
+- Use the following link for "$ git clone" the repo:
+https://github.com/Vitor-Guedes-Madeira/Capstone-React/
+- cd Capstone-React
+- "npm install" to install all the dependencies
+- "npm start" to run the live version locally
+- "code ." To open the project on your local editor
+
+- Please click on the "acess link" so you can view the live demo
+- You can use it for functional math-related quotes and calculation
+
+## Author
+
+👤 **Vitor Guedes Madeira**
+- GitHub: [@vitorguedesmadeira](https://github.com/VitorGuedesMadeira)
+- Twitter: [@CDahguedes](https://twitter.com/CDahguedes)
+- LinkedIn: (https://www.linkedin.com/in/vitor-guedes-madeira/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+Feel free to follow me and share any feedback!
+
+## Acknowledgments
+
+- Marvel Studios oficial API
+- Hat tip to anyone whose code was used
+- Especial thanks to my friends Mumenya, Raul and Luis who are always open to give me feedbacks
+
+## Design Template
+
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
```

Comment:
- [ ] [Optional] [Highly recommended] Please add this line to the `Acknowledgments` section above to make your redme file more professional.
[Hint] You can add it in this form:
I give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))

### [Highlight] Once again like the last week this design is one of the best designs I reviewed for this project, So Keep up the good work 💪.

Reply:
Thanks for the tip! I've already implemented it! ✔️ 😄